javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,10 @@
+
+const weekday = ['Sunday', 'Monday', 'Tuseday', 'Wendsday', 'Thursday','Friday','Saturday']
+function getEventWeekday (number) {
```

Comment:
An alternative name for the parameter could be `daysUntilEvent` which I find aligns better with the homework specification:
> You specify how many days from today an event is being held.

I want to emphasize that `number` is not wrong to use :smile: